haystack/utils/semantic_answer_similarity.py

```diff
@@ -0,0 +1,101 @@
+from typing import List, Tuple, Optional, Union
+from transformers import AutoConfig
+from sentence_transformers import SentenceTransformer, CrossEncoder
+from sklearn.metrics.pairwise import cosine_similarity
+import numpy as np
+
+
+def semantic_answer_similarity(
```

Comment:
I think it makes more sense to move semantic_answer_similarity to `haystack/modeling/evaluation/metrics.py` than to `utils`. If you move it there and update the import in `haystack/pipelines/base.py` accordingly, I'll happily approve this PR. 👍